src/main/webapp/about.jsp

```diff
@@ -24,12 +24,20 @@
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
-    <% if(request.getSession().getAttribute("user") != null){ %>
+        <% if(request.getSession().getAttribute("user") != null){ %>
       <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+        <%if(request.getSession().getAttribute("user").equals("EmilyArroyo")| request.getSession().getAttribute("user").equals("AlexandriaStorm")|request.getSession().getAttribute("user").equals("AnaVitoriadoValleCosta")|request.getSession().getAttribute("user").equals("KevinWorkman")|request.getSession().getAttribute("user").equals("GavinLifrieri")) { %>
```

Comment:
These are just small things, but I would probably use `||` instead of `|`. There's a subtle difference between the two: `||` will stop calculations as soon as it finds a `true`, whereas `|` will check every single thing in the `if` statement, even if it doesn't need to.

Also keep in mind that everybody should register with these usernames whenever we deploy to the live site. Do you want to take lead on reminding everybody about that?

Reply:
Sure, I will send a message today to the team about this. And okay, It wasn't happy earlier when I used || but will try that again!